pretrain/tinyllama.py

```diff
@@ -24,21 +24,24 @@
 from torchmetrics.aggregation import RunningMean
 from typing_extensions import Literal
 
+
 # support running without installing as a package
 wd = Path(__file__).parent.parent.resolve()
 sys.path.append(str(wd))
 
 from lit_gpt.args import EvalArgs, IOArgs, TrainArgs
 from lit_gpt.model import GPT, Block, CausalSelfAttention, Config, LLaMAMLP
 from lit_gpt.utils import CLI, CycleIterator, chunked_cross_entropy, num_parameters
+from lit_gpt.datasets import TinyLlama, LitDataModule
 
 
 def setup(
-    model_name: str = "tiny-llama-1.1b",
+    model: Config = Config(name="tiny-llama-1.1b"),
     name: str = "lit-tiny-llama-1.1b",
```

Comment:
Remove this argument? It's already included in the config and in io.out_dir. We could take it from one of them

Reply:
agreed, removed